.github/workflows/conda.yml

```diff
@@ -96,16 +97,17 @@ jobs:
           - {os: ubuntu-24.04, os-family: linux-64}
           - {os: windows-2019, os-family: win-64}
           - {os: windows-2022, os-family: win-64}
-          - {os: macos-12, os-family: osx-64}
```

Comment:
On ne build plus pour macos-12, parce que github ne le permet plus?
- cette version est-elle vraiment obsolète?
  - sur wikipedia, on indique que la dernière version stable date du 29 juillet 2024 
    cf. https://fr.wikipedia.org/wiki/MacOS_Monterey
- y-a-t-il possiblement des utilisateurs pour cette version?
- y-a-t-il une façon alternative de construire cette version?
- attention à mettre à jour le site web sur les version supportées au moment de la diffusion de la 10.2.4

Reply:
Yes, MacOS 12 is really not supported anymore; see. e.g. https://github.com/KhiopsML/khiops/actions/runs/12164968193. And also https://github.com/actions/runner-images/issues/10721: MacOS12 is not supported anymore since the 3rd of December, 2024.

Regarding MacOS 15: we had already done the CI updates on the Khiops Python side for the aborted 10.2.3.1 version. So, I see no problem in testing MacOS 15 in the CI. We could officially support it once we are able to fully test this target platform.